### PR TITLE
feature: display user's own ratings at the top of the 'recent ratings' listing

### PR DESCRIPTION
**Pull request by x2corp**

### CODE REVIEW

**Comment by x2corp** [comment]
ignore, pushed early

File: init.sql [hunk 1]
```diff
@@ -122,7 +122,7 @@ CREATE TABLE `ratings` (
   `BeatmapID` int NOT NULL,
   `UserID` int NOT NULL,
   `Score` decimal(2,1) DEFAULT NULL,
-  `date` datetime NOT NULL ON UPDATE CURRENT_TIMESTAMP,
+  `date` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
```
Comment (apollo-dw) [inline init.sql:125]: @x2corp I know this is closed, but this should stay as "ON UPDATE" -- we want to bump up ratings when users update them.

Comment (x2corp) [reply init.sql:125]: @apollo-dw I feel like this goes against the behavior of re-scaling/re-rating though? Maybe a toggle in user settings would be suitable for this?

I'm not just sure people would want the timestamp to be updated for moving it down .5 or anything of the sort.

Comment (apollo-dw) [reply init.sql:125]: @x2corp It helps with the recent display on the front page to show activity on the site though. Maybe it'd be good to have "created at" and "updated at" fields...

**Pull request closed without merging**